docs/api/application_overview.md

```diff
@@ -0,0 +1,49 @@
+# Application overview
+
+The application is setup using a layered architecture (see [Link](architecture.md))  
+To create the application you will need to create these three components: logger, data-access layer and model builder (see [main.py](https://github.com/Deltares/D-EcoImpact/blob/main/main.py)).  
+
+```python
+    # configure logger and data-access layer
+    logger: ILogger = LoggerFactory.create_logger()
+    da_layer: IDataAccessLayer = DataAccessLayer(logger)
+    model_builder = ModelBuilder(da_layer, logger)
+
+    # create and run application
+    application = Application(logger, da_layer, model_builder)
+    application.run(path)
```

Comment:
What is the variable "path"? I know this is not meant to be runnable code, but still, to understand what is happening it would be nice to know this.

Reply:
This is the path to the input file. This code comes from the main.py code that is added as a link